iosApp/PhongKMMIC/Sources/Presentation/Modules/Home/Views/HomeHeaderView.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  HomeHeaderView.swift
+//  PhongKMMIC
+//
+//  Created by Phong Vo on 07/05/2023.
+//  Copyright © 2023 Nimble. All rights reserved.
+//
+
+import SwiftUI
+
+struct HomeHeaderView: View {
+
+    var body: some View {
+        HStack {
+            VStack(spacing: 10.0) {
+                Text("MONDAY, JUNE 15")
+                    .foregroundColor(.white)
+                    .font(.boldSmall)
+                Text("TODAY")
```

Comment:
We can move this `TODAY` text to `Localizable.strings` 🙋

Reply:
solved in 149e645